packages/fx-core/src/component/driver/teamsApp/interfaces/ValidateTeamsAppArgs.ts

```diff
@@ -3,7 +3,11 @@
 
 export interface ValidateTeamsAppArgs {
   /**
-   * Teams app manifest template path
+   * Teams app manifest path
    */
-  manifestPath: string;
+  manifestPath?: string;
```

Comment:
Seems this property is not used, is it OK to remove?

Reply:
We will add it back when VSCode UI has been added.